.github/workflows/main.yml

```diff
@@ -24,11 +24,11 @@ jobs:
           nodeenv -p && \
           npm install -g npm-check-updates && \
           find ./docker -name package.json -exec ncu --packageFile {} -u \;
-      - name: Push to main branch
-        run: |
-          git config --global user.name "xmedr"
-          git config --global user.email "xavier.medrano@datamade.us"
+      # - name: Push to main branch
```

Comment:
automated updating of requirements go us to a point where we these cookie cutters were in a broken state. let's pause until we can figure out a better way to upgrade deps.